README.md

```diff
@@ -34,6 +34,25 @@ browser.addCommand('getOcrText', command('POST', '/session/:sessionId/appium/ocr
 }))
 ```
 
+Also for reference, here is how to add the command to the python-appium-client:
+```py
+class CustomURLCommand(ExtensionBase):
```

Comment:
let's call this `OCRCommand` so it's clear what it's related to

Reply:
Agreed